extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -857,10 +857,15 @@ export class ProjectsController {
 
 		const newFilePath = path.join(path.dirname(utils.getPlatformSafeFileEntryPath(file?.relativePath!)), `${newFileName}.sql`);
 
-		try {
-			await this.move(node, node.projectFileUri.fsPath, newFilePath);
-		} catch (e) {
```

Comment:
these try catches were from before the swap to use the STS api, when something could have gone wrong in ADS. Now that this uses the move api in STS, if something goes wrong in DacFx or STS, it will be indicated in the ResultStatus returned from ADS.